packages/utilities/src/styled.tsx

```diff
@@ -5,7 +5,7 @@ import { IStyleFunction } from './IStyleFunction';
 import { CustomizableContextTypes } from './customizable';
 import { Customizations, ICustomizations } from './Customizations';
 
-export type IStyleFunctionOrObject<TStyleProps, TStyles> = IStyleFunction<TStyleProps, TStyles> | TStyles;
+export type IStyleFunctionOrObject<TStyleProps, TStyles> = IStyleFunction<TStyleProps, TStyles> | Partial<TStyles>;
```

Comment:
so is best practice to make all of the IComponentStyles required or optional?

![image](https://user-images.githubusercontent.com/1434956/41559470-f28d5cc8-72f8-11e8-8f51-471c9fe14dbc.png)

Seems with this change it wouldn't matter.

Reply:
It is consistent with IStyleFunction now.. but I agree, now it doesn't matter how IPersonaStyles is specified and we could remove optional from each individual parameter.

Reply:
I already changed the wiki back :D